core/src/models/Destination.ts

```diff
@@ -84,19 +113,19 @@ export const DestinationSyncModeData: Record<
       delete: true,
     },
   },
-  additive: {
-    key: "additive",
-    displayName: "Additive",
+  upsert: {
+    key: "upsert",
+    displayName: "Upsert",
     description: "Sync all records, but do not delete (create, update)",
```

Comment:
I'm thinking we should also update these descriptions away from the word "sync". Any suggestions?

Reply:
`Export` perhaps?